openshift/e2e-tests-openshift.sh

```diff
@@ -17,10 +17,14 @@ readonly TEST_ORIGIN_CONFORMANCE="${TEST_ORIGIN_CONFORMANCE:-"false"}"
 readonly SERVING_NAMESPACE=knative-serving
 readonly EVENTING_NAMESPACE=knative-eventing
 readonly TARGET_IMAGE_PREFIX="$INTERNAL_REGISTRY/$EVENTING_NAMESPACE/knative-eventing-"
-# TODO: Subscription.spec.sourceNamespace does not work on OCP v4.2. Need to revert after https://jira.coreos.com/browse/OLM-1190 is solved.
-readonly OLM_NAMESPACE_SERVING="knative-serving"
-readonly OLM_NAMESPACE_EVENTING="knative-eventing"
-#readonly OLM_NAMESPACE="openshift-operator-lifecycle-manager"
+
+# The OLM global namespace was moved to openshift-marketplace since v4.2
+# ref: https://jira.coreos.com/browse/OLM-1190
+if [ ${HOSTNAME} = "e2e-aws-ocp-41" ]; then
```

Comment:
hm, what about the case where `${HOSTNAME}` is just "e2e-aws"?

Reply:
e2e-aws-ocp-41 is what we did it set up to . 

I did borrow this from @nak3 :-)